### DISCUSSION TITLE
百度令牌获取？

### DISCUSSION THREAD

**Discussion by asjusdjkfd** [AlistGo/alist#7629] [Q&A] [answered]
以下是报错信息，不确定是不是我的网络问题
Get "https://openapi.baidu.com/oauth/2.0/token?grant_type=authorization_code&code=e8e781c9bd4ef25b8aa52b10c96681f5&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&client_secret=jXiFMOPVPCWlO2M5CwWQzffpNPaGTRBG&redirect_uri=https://alist.nn.ci/tool/baidu/callback": dial tcp 39.156.66.111:443: i/o timeout

**Comment by ziyu499** [comment]
请问你解决了吗，我这边也是同样的问题

**Reply by asjusdjkfd** [reply, marked as answer]
现在可以获取了
